### ISSUE TITLE
Datastream dataset_fields_ids are getting sorted numerically - forces replacement

### ISSUE THREAD

**Issue by mikecarrington** [akamai/terraform-provider-akamai#263]
Hi

Seems that the Datastream resource dataset_fields_ids set is getting sorted numerically.
This results in an update every time you run terraform apply, which is unnecessary.

Appreciate datastream capability has only recently been released.

### Terraform Version
➜ terraform --version
Terraform v1.0.9
on linux_amd64
+ provider registry.terraform.io/akamai/akamai v1.8.0

### Affected Resource(s)
Please list the resources as a list, for example:
- akamai_datastream

### Terraform Configuration Files
```hcl
resource "akamai_datastream" "stream" {
    active             = false
    config {
        format             = "JSON"
        frequency { time_in_sec = 60}
        upload_file_prefix = "ds2"
        upload_file_suffix = "ds"
    }
    contract_id        = var.contract_id
    dataset_fields_ids = [ 1002,1100,1006,1008,1011,1012,1013,1017,2008,2009,1032,1033,1068,1102,1103,2007,2010 ]
    email_ids          = var.email_ids
    group_id           = var.group_id
    property_ids       = [ var.property_id ]
    stream_name        = var.stream_name
    stream_type        = "RAW_LOGS"
    template_name      = "EDGE_LOGS"
    s3_connector {    }
}
```
note: s3_connector content removed for brevity.

### Expected Behavior
Sort order of the `dataset_fields_ids` should be maintained.

### Actual Behavior
If you submit the following values:
`dataset_fields_ids = [ 1002,1100,1006,1008,1011,1012,1013,1017,2008,2009,1032,1033,1068,1102,1103,2007,2010 ]`

The terraform plan will show:
```
      + dataset_fields_ids = [
          + 1002,
          + 1006,
          + 1008,
          + 1011,
          + 1012,
          + 1013,
          + 1017,
          + 1032,
          + 1033,
          + 1068,
          + 1100,
          + 1102,
          + 1103,
          + 2007,
          + 2008,
          + 2009,
          + 2010,
        ]

```
On the next `terraform apply` you will see an unnecessary update to the DS config:
```
  # module.akamai_datastream["name"].akamai_datastream.stream will be updated in-place
  ~ resource "akamai_datastream" "stream" {
      ~ dataset_fields_ids = [
            1002,
          - 1100,
            1006,
            # (4 unchanged elements hidden)
            1017,
          - 2008,
          - 2009,
            1032,
            # (1 unchanged element hidden)
            1068,
          + 1100,
            1102,
            # (1 unchanged element hidden)
            2007,
          + 2008,
          + 2009,
            2010,
        ]
        id                 = "13587"
        # (17 unchanged attributes hidden)
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

Many thanks
Mike

**Comment by mikecarrington** [comment]
Just as a follow up - the additional behaviour we're seeing is that as the duration of the `terraform apply` (to activate the new config) is often > 1.5 hours the process times out and the resource is marked as tainted.

```
# module.akamai_datastream["name"].akamai_datastream.stream is tainted, so must be replaced
```

When next applying an update as its tainted the resource is getting destroyed first and then recreated - which could cause a loss of log data for the duration of the lifecycle from de-activation through to activation.

**Comment by lkowalsk-akamai-com** [comment]
Hi @mikecarrington,
Thank you for using Datastream subprovider and sharing the feedback. We will address both of the issues you've outlined in next Terraform Provider version which will be releasesd in next couple of weeks.

**Comment by lkowalsk-akamai-com** [comment]
Hi @mikecarrington 
This issue is now fixed with relese v1.9.0 of Akamai Terraform Provider. I'm closing this issue now. Plesae feel free to reaopen the issue if you find it not fixed.

**Comment by mikecarrington** [comment]
Many thanks :+1:
